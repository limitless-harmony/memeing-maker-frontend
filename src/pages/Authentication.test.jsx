import React from 'react';
import { shallow } from 'enzyme';

import AuthComponent from 'pages/Authentication';

describe('Testing the Authentication component', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<AuthComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
