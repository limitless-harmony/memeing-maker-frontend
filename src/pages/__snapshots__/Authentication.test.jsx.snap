// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Authentication component renders without crashing 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
